README.md

```diff
@@ -104,6 +103,28 @@ cornac.Experiment(eval_method=rs, models=models, metrics=metrics, user_based=Tru
 For more details, please take a look at our [examples](examples) as well as [tutorials](tutorials). For learning purposes, this list of [tutorials on recommender systems](https://github.com/PreferredAI/tutorials/tree/master/recommender-systems) will be more organized and comprehensive. 
 
 
+## Simple model serving
+
+Here, we provide a simple way to serve a Cornac model by launching a standalone web service. While this will not be an optimized service for model deployment in production, it is quite handy for testing or creating a demo application. Supposed that we use the trained BPR from previous example, we first need to save the model:
+```python
+bpr.save("save_dir")
+```
+The model can be deployed easily by triggering Cornac serving module:
+```bash
+$ python -m cornac.serving --model_dir save_dir/BPR --model_class cornac.models.BPR
+
+# Serving BPR at port 8080
+```
+Here you go, your model service is now ready. Let's get `top-5` item recommendations for the user `"63"`:
+```bash
+$ curl http://127.0.0.1:8080/recommend \
```

Comment:
Please consider using GET request instead of using POST for this purpose. We can also use request query such as `?uid=63&k=5` in replacement of request body.

Reply:
Thanks for the comment. The reason we put data in the json body is because `uid` is raw string ID of user in the original data, not the integer index. We just try to avoid any encoding issue for cases where `uid` contains special characters/white spaces. With that said, those cases are rare and we can consider switching to GET request.

Reply:
@darrylong @hieuddo what do you guys think?

Reply:
Cool. I think we can develop more user-friendly queries in the future. Is that the further goal, or only provide API?

Reply:
@tqtg Since we are obtaining recommendations, I think GET seems more appropriate just not to complicate the developers. I think string `uid` should work as well in GET, unless there are unique formats?

Reply:
I've changed to GET request. We can change to POST later if we face real encoding issue.